app/components/SimulationViewer/SimulationViewer.tsx

```diff
@@ -40,33 +42,65 @@ export default function SimulationViewer({
   };
 
   const ruleMapInputs = createRuleMap(rulemap.inputs);
-  const ruleMapOutputs = createRuleMap(rulemap.outputs);
   const ruleMapResultOutputs = createRuleMap(rulemap.resultOutputs);
 
-  const [selectedSubmissionInputs, setSelectedSubmissionInputs] = useState<SubmissionData>(ruleMapInputs);
-  const [contextToSimulate, setContextToSimulate] = useState<SubmissionData | null>();
-  const [outputSchema, setOutputSchema] = useState<Record<string, any> | null>(ruleMapOutputs);
```

Comment:
I noticed the data for this wasn't actually being used anywhere so I removed it and it's subsequent function calls that set the data. Don't know if it was meant to be using for something?

Reply:
The things this data was used for are no longer present and have been supplanted. Thank you for removing!